.clang-tidy

```diff
@@ -8,6 +8,7 @@ Checks:
   -modernize-avoid-c-arrays,
   -modernize-use-trailing-return-type,
   -modernize-concat-nested-namespaces,
+  -modernize-use-default-member-init,
```

Comment:
Suppressed for now as -fix doesn't deal well with macro conditionals and produces incorrect code.